src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8840,10 +8800,6 @@ private bool IsPossibleStatement(bool acceptAccessibilityMods)
                 case SyntaxKind.IdentifierToken:
                     return IsTrueIdentifier();
 
-                case SyntaxKind.CatchKeyword:
-                case SyntaxKind.FinallyKeyword:
-                    return !_isInTry;
```

Comment:
on the fence... this def can be called on those (this is IsPossibleStatement).  but explicitly hitting them and stating that they are not statements (which is what we want so that higher up try-statements actually will consume error catches/finallys into them), seems inverted from how this method generally works.  specifically that it says the cases that def are statements, not hte cases that are not.

I would be ok with having these there commented otu saying why we explicitly do not consider them the start of statements.

Reply:
Let's leave as-is then